src/Header/Header.jsx

```diff
@@ -1,71 +1,144 @@
-import React from "react";
+import React, { useState } from "react";
+
+import cx from "classnames";
 
 import { Link } from "react-router-dom";
 import { Hidden, Visible } from "react-grid-system";
 import Logo from "../assets/godragons_logo_white_text-2.png";
 import "./Header.scss";
 
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
 import BoxedLayout from "../Common/BoxedLayout/BoxedLayout";
 
 function Header() {
-  function displayDesktopNav() {
+  const [mobileNavIsOpen, setMobileNavIsOpen] = useState(false);
+  const [
+    mobileNavBackgroundIsShowing,
+    setMobileNavBackgroundIsShowing
+  ] = useState(false);
+
+  function displayDesktopHeader() {
     return (
-      <>
+      <div className="desktop-header">
         <div className="logo-container">
           <Link to="/">
             <img className="logo" src={Logo} alt="godragons logo" />
           </Link>
         </div>
-        <div className="nav-container">
-          <nav className="main-menu">
-            <ul>
-              <li>
-                <Link to="/case-studies">Case Studies</Link>
-              </li>
-              <li>
-                <Link to="/services">Services</Link>
-              </li>
-              <li>
-                <Link to="/pricing">Pricing</Link>
-              </li>
-              <li>
-                <Link to="/team">Team</Link>
-              </li>
-              <li>
-                <Link to="/contact">Contact</Link>
-              </li>
-              <li>
-                <Link to="/blog">Blog</Link>
-              </li>
-            </ul>
-          </nav>
-        </div>
-      </>
+        {displayNav()}
+      </div>
     );
   }
 
-  function displayMobileNav() {
+  function displayMobileHeader() {
     return (
-      <>
+      <div className="mobile-header">
         <div className="logo-container">
           <Link to="/">
             <img className="logo" src={Logo} alt="godragons logo" />
           </Link>
         </div>
-      </>
+        <div className="hamburger-icon" onClick={toggleMobileNav}>
```

Comment:
isn't the onclick just `showMobileNav`? do we even need the `toggleMobileNav` function?

Reply:
In the current configuration, you're right. Initially, it looked different.